x/ethbridge/types/querier.go

```diff
@@ -10,15 +10,23 @@ import (
 // QueryEthProphecyParams defines the params for the following queries:
 // - 'custom/ethbridge/prophecies/'
 type QueryEthProphecyParams struct {
-	Nonce          int             `json:"nonce"`
-	EthereumSender EthereumAddress `json:"ethereum_sender"`
+	EthereumChainID       int             `json:"ethereum_chain_id"`
+	BridgeContractAddress EthereumAddress `json:"bridge_contract_address"`
+	Nonce                 int             `json:"nonce"`
+	Symbol                string          `json:"symbol"`
+	TokenContractAddress  EthereumAddress `json:"token_contract_address"`
+	EthereumSender        EthereumAddress `json:"ethereum_sender"`
 }
 
 // QueryEthProphecyParams creates a new QueryEthProphecyParams
-func NewQueryEthProphecyParams(nonce int, ethereumSender EthereumAddress) QueryEthProphecyParams {
+func NewQueryEthProphecyParams(ethereumChainID int, bridgeContractAddress EthereumAddress, nonce int, symbol string, tokenContractAddress EthereumAddress, ethereumSender EthereumAddress) QueryEthProphecyParams {
```

Comment:
```suggestion
func NewQueryEthProphecyParams(ethereumChainID int, bridgeContractAddress EthereumAddress, nonce int, symbol string, tokenContractAddress, ethereumSender EthereumAddress) QueryEthProphecyParams {
```

Reply:
Why are we removing type EthereumAddress here?